src/Raven.Studio/typescript/viewmodels/database/documents/setCounterDialog.ts

```diff
@@ -0,0 +1,55 @@
+import dialogViewModelBase = require("viewmodels/dialogViewModelBase");
+import database = require("models/resources/database")
+import setCounterCommand = require("commands/database/documents/counters/setCounterCommand");
+
+
+class setCounterDialog extends dialogViewModelBase {
+    
+    // todo.. add validation on input..
+    
+    result = $.Deferred<void>();
+
+    createNewCounter = ko.observable<boolean>();
+    counterName = ko.observable<string>();
+    
+    totalValue = ko.observable<number>();
+    newTotalValue = ko.observable<number>();
+    
+    counterValuesPerNode = ko.observableArray<nodeCounterValue>();
+
+    spinners = {
+        update: ko.observable<boolean>(false)
+    };    
+    
+    compositionComplete() {
```

Comment:
no need for this method - as it is calling parent.

Reply:
done